src/isIsogram.test.js

```diff
@@ -3,7 +3,37 @@
 describe('isIsogram', () => {
   const { isIsogram } = require('./isIsogram');
 
-  it(`should be declared`, () => {
-    expect(isIsogram).toBeInstanceOf(Function);
+  describe('`isIsogram` function', () => {
```

Comment:
There should be 1 describe with all the tests (1 describe = 1 test suite, 1 it = 1 test case)